### PR TITLE
Update the default net to nn-76a8a7ffb820.nnue.

### PR DESCRIPTION
**Pull request by MichaelB7**
First, I  tip my hat and express my gratittude for all the great work Sergio Vieri has done in helping transforming Stockfish to a premier neural net chess engine.  I am proud and honored to be in a position to submit this pull request on behalf of Sergio Vieri , this is really his net.  The net being submitted was   trained with the pytorch trainer: https://github.com/glinscott/nnue-pytorch

python train.py i:/bin/all.binpack i:/bin/all.binpack --gpus 1 --threads 4 --num-workers 30 --batch-size 16384 --progress_bar_refresh_rate 30 --smart-fen-skipping --random-fen-skipping 3 --features=HalfKAv2^ --auto_lr_find True --lambda=1.0 --max_epochs=240 --seed %random%%random% --default_root_dir exp/run_109 --resume-from-model ./pt/nn-d8609abe8caf.pt

This run is thus started from Segio Vieri's net [nn-d8609abe8caf.nnue](https://tests.stockfishchess.org/api/nn/nn-d8609abe8caf.nnue)

all.binpack equaled 4 parts Wrong_NNUE_2.binpack https://drive.google.com/file/d/1seGNOqcVdvK_vPNq98j-zV3XPE5zWAeq/view?usp=sharing plus two parts of Training_Data.binpack https://drive.google.com/file/d/1RFkQES3DpsiJqsOtUshENtzPfFgUmEff/view?usp=sharing
Each set was concatenated together - making one large Wrong_NNUE 2 binpack and one large Training so the were approximately equal in size. They were then interleaved together. The idea was to give Wrong_NNUE.binpack closer to equal weighting with the Training_Data binpack

model.py modifications: 
    loss = torch.pow(torch.abs(p - q), 2.6).mean()
   LR = 8.0e-5  calculated as follows: 1.5e-3*(.992^360) - the idea here was to take a highly trained net and just use all.binpack as a finishing micro refinement touch for the last 2 Elo or so.  This net was discovered on the 59th epoch.
      optimizer = ranger.Ranger(train_params, betas=(.90, 0.999), eps=1.0e-7, gc_loc=False, use_gc=False)
    # Drop learning rate after 75 epochs
    scheduler = torch.optim.lr_scheduler.StepLR(optimizer, step_size=1, gamma=0.992)
   For this micro optimization, I had set the period to "5" in train.py.  This changes the checkpoint output so that every 5th checkpoint file is created 

By the way, the original LTC [test](https://tests.stockfishchess.org/html/live_elo.html?60f9e733d8a6b65b2f3a7770) does not appear as strong and the final touches were to adjust the NNUE scale.  A technique I learned by watching @SFisGOD fishtests.

passed LTC
https://tests.stockfishchess.org/tests/view/60fa45aed8a6b65b2f3a77a4
LLR: 2.94 (-2.94,2.94) <0.50,3.50>
Total: 53040 W: 1732 L: 1575 D: 49733
Ptnml(0-2): 14, 1432, 23474, 1583, 17

passed STC
https://tests.stockfishchess.org/tests/view/60f9fee2d8a6b65b2f3a7775
LLR: 2.94 (-2.94,2.94) <-0.50,2.50>
Total: 37928 W: 3178 L: 3001 D: 31749
Ptnml(0-2): 100, 2446, 13695, 2623, 100.

Bench: 5169957

### CODE REVIEW

**Comment by SFisGOD** [comment]
@MichaelB7 Well done!

However, I suggest to hold this as we wait if this will pass https://tests.stockfishchess.org/tests/view/60fb8929d8a6b65b2f3a7885
It will be weird to have an older net as master (greenlighted second)

**Comment by MichaelB7** [comment]
@SFisGOD Thanks - yes I agree.  Make sure we get the best net!

**Comment by sergiovieri** [comment]
The initial net [nn-d8609abe8caf.nnue](https://tests.stockfishchess.org/api/nn/nn-d8609abe8caf.nnue) is trained by generating around 16B of training data from the last master net [nn-9e3c6298299a.nnue](https://tests.stockfishchess.org/api/nn/nn-9e3c6298299a.nnue), then trained, continuing from the master net, with lambda=0.2 and sampling ratio of 1. Starting with LR=2e-3, dropping LR with a factor of 0.5 until it reaches LR=5e-4. `in_scaling` is set to 361. No other significant changes made to the pytorch trainer.

Training data gen command (generates in chunks of 200k positions):
```
generate_training_data min_depth 9 max_depth 11 count 200000 random_move_count 10 random_move_max_ply 80 random_multi_pv 12 random_multi_pv_diff 100 random_multi_pv_depth 8 write_min_ply 10 eval_limit 1500 book noob_3moves.epd output_file_name gendata/$(date +"%Y%m%d-%H%M")_${HOSTNAME}.binpack
```

PyTorch trainer command (Note that this only trains for 20 epochs, repeatedly train until convergence):
```
python train.py --features "HalfKAv2^" --max_epochs 20 --smart-fen-skipping --random-fen-skipping 500 --batch-size 8192 --default_root_dir $dir --seed $RANDOM --threads 4 --num-workers 32 --gpus $gpuids --track_grad_norm 2 --gradient_clip_val 0.05 --lambda 0.2 --log_every_n_steps 50 $resumeopt $data $val
```

See [sergiovieri:tools_mod](https://github.com/sergiovieri/Stockfish/tree/tools_mod/rl) for the scripts used to generate data.
